style.css

```diff
@@ -3,89 +3,360 @@
 * {
   margin: 0;
   padding: 0;
+  font-family: "poppins", sans-serif;
 }
 
-.nav-bar {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
+body {
+  background-color: rgba(229, 229, 229, 0.3);
 }
 
-.logo {
-  margin-top: 58px;
-  margin-left: 24px;
-}
+@media only screen and (max-width: 600px) {
```

Comment:
- [optional] please set the max width to 768px which is the start of the desktop version. this will make your work responsive to that stage
![Screenshot (1024)](https://user-images.githubusercontent.com/65298048/235297080-a932a3ee-74be-48fa-a05e-d8c8a383170d.png)